packages/next/taskfile-ncc.js

```diff
@@ -41,6 +41,22 @@ function writePackageManifest (packageName) {
   let typesFile = types || typings
   if (typesFile) {
     typesFile = require.resolve(join(packageName, typesFile))
+  } else {
+    try {
+      const typesPackage = `@types/${packageName}`
+
+      const { types, typings } = require(join(typesPackage, `package.json`))
+      typesFile = types || typings
```

Comment:
```suggestion
      typesFile = types || typings || 'index'
```

Should this default to `index` it looks like `@types/resolve` doesn't have a `types` or `typings` field

Reply:
When that default behavior was enabled it started conflicting with our existing types. I'd rather do this change in a follow-up PR if that's OK?

Reply:
👌